### ISSUE TITLE
[Linux] [2.2.2] last sync connection with the client after restart is deleted

### ISSUE THREAD

**Issue by zoom61** [owncloud/client#5098]
### Expected behavior
The new connection should be visible on restart

### Actual behavior
The connection is not in the owncloud-client

### Steps to reproduce
1. I have three cloud works
2. The latter is not preserved in Linux reboot

### Client configuration
Client version: Version 2.2.2
Operating system: Kubuntu 16.04
OS language: French
Installation path of client: default

**Comment by guruz** [comment]
Which Qt version is your client using?
As far as I know there had been settings problems with Qt 4.x


CC @ogoffart

**Comment by zoom61** [comment]
My Qt version is 5.5.1.
But it works on other PCs that have a cloud connection. In my case, I add a new connection, and it is not stored on restart. While this works when I install it.

**Comment by guruz** [comment]
@zoom61 What about a logfile? ```owncloud --logfile -```.
Do you see anything interesting there?
Did you check ```owncloud.cfg``` after adding the connection?

**Comment by zoom61** [comment]
Once the installation done, the `owncloud.cfg` file contains no new information.
I have added manually, and it works correctly.

**Comment by guruz** [comment]
@zoom61 You mean when you added in UI you can confirm that afterwards it was not in ```owncloud.cfg```?

**Comment by zoom61** [comment]
Yes. When I added a new connexion, the `owncloud.cfg` isn't modify.

**Comment by hodyroff** [comment]
Does it work in the newest client? Hope we can close here ...

**Comment by eduardoramalho** [comment]
Apparently I have the same problem but I have 2 "same" computers (it have the same configuration and installation, just some hardware differences) but only one has the bug.

There are 2 Ubuntu "Release 18.04.3 LTS (Bionic Beaver) 64-bit". Every time I shut down or restart the "computer 1", it removes the user connection (I have only 1 user, I will test add 2 now to see if it's only the last one that is removed).

My owncloud.cfg after connecting to "computer 1" is this:
"[General]
clientVersion = 2.5.4 (build 515)
optionalDesktopNotifications = true

[Accounts]
version = 2 "

On "Computer 2", which does not display the error is as follows:

[General]
clientVersion=2.5.4 (build 515)
confirmExternalStorage=true
newBigFolderSizeLimit=500
optionalDesktopNotifications=true
useNewBigFolderSizeLimit=true

[Accounts]
0\Folders\1\ignoreHiddenFiles=true
0\Folders\1\journalPath=._sync_970cd8d49433.db
0\Folders\1\localPath=/home/devmyuser/ownCloud/
0\Folders\1\paused=false
0\Folders\1\targetPath=/
0\Folders\1\usePlaceholders=false
0\Folders\1\version=1
0\authType=http
0\dav_user=myuser1
0\http_oauth=false
0\http_user=myuser1
0\serverVersion=10.0.10.4
0\url=https://myurl.changednowforprivacy
0\user=myuser1
0\version=1
version=2

[Settings]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x3\0\0\0\0\x1T\0\0\0\x1f\0\0\x4\n\0\0\x2\x17\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16\0\0\0\x1\0\0\0\0\x5P\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16)

**Comment by eduardoramalho** [comment]
> Apparently I have the same problem but I have 2 "same" computers (it have the same configuration and installation, just some hardware differences) but only one has the bug.
> 
> There are 2 Ubuntu "Release 18.04.3 LTS (Bionic Beaver) 64-bit". Every time I shut down or restart the "computer 1", it removes the user connection (I have only 1 user, I will test add 2 now to see if it's only the last one that is removed).
> 
> My owncloud.cfg after connecting to "computer 1" is this:
> "[General]
> clientVersion = 2.5.4 (build 515)
> optionalDesktopNotifications = true
> 
> [Accounts]
> version = 2 "
> 
> On "Computer 2", which does not display the error is as follows:
> 
> [General]
> clientVersion=2.5.4 (build 515)
> confirmExternalStorage=true
> newBigFolderSizeLimit=500
> optionalDesktopNotifications=true
> useNewBigFolderSizeLimit=true
> 
> [Accounts]
> 0\Folders\1\ignoreHiddenFiles=true
> 0\Folders\1\journalPath=._sync_970cd8d49433.db
> 0\Folders\1\localPath=/home/devmyuser/ownCloud/
> 0\Folders\1\paused=false
> 0\Folders\1\targetPath=/
> 0\Folders\1\usePlaceholders=false
> 0\Folders\1\version=1
> 0\authType=http
> 0\dav_user=myuser1
> 0\http_oauth=false
> 0\http_user=myuser1
> 0\serverVersion=10.0.10.4
> 0\url=https://myurl.changednowforprivacy
> 0\user=myuser1
> 0\version=1
> version=2
> 
> [Settings]
> geometry=@bytearray(\x1\xd9\xd0\xcb\0\x3\0\0\0\0\x1T\0\0\0\x1f\0\0\x4\n\0\0\x2\x17\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16\0\0\0\x1\0\0\0\0\x5P\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16)

-------------------------------Updating:-------------------------------
owncloud.cfg on "computer 1" with two users:
[General]
clientVersion=2.5.4 (build 515)
optionalDesktopNotifications=true

[Accounts]
version=2

owncloud.cfg on "computer 2"(wich does not have the error), with 2 users:
[General]
clientVersion=2.5.4 (build 515)
confirmExternalStorage=true
newBigFolderSizeLimit=500
optionalDesktopNotifications=true
useNewBigFolderSizeLimit=true

[Accounts]
0\Folders\1\ignoreHiddenFiles=true
0\Folders\1\journalPath=._sync_970cd8d49433.db
0\Folders\1\localPath=/home/SecretName/ownCloud/
0\Folders\1\paused=false
0\Folders\1\targetPath=/
0\Folders\1\usePlaceholders=false
0\Folders\1\version=1
0\authType=http
0\dav_user=SecretName1
0\http_oauth=false
0\http_user=SecretName1
0\serverVersion=10.0.10.4
0\url=https://SecretName.site
0\user=SecretName1
0\version=1
1\Folders\2\ignoreHiddenFiles=true
1\Folders\2\journalPath=._sync_22ac2a3fd6db.db
1\Folders\2\localPath=/var/www/
1\Folders\2\paused=false
1\Folders\2\targetPath=/
1\Folders\2\usePlaceholders=false
1\Folders\2\version=1
1\authType=http
1\dav_user=SecretName2@SecretSite.notAClue
1\http_oauth=false
1\http_user=SecretName2@SecretSite.notAClue
1\serverVersion=10.0.10.4
1\url=https://SecretName.site
1\user=SecretName2@SecretSite.notAClue
1\version=1
version=2

[ActivityListHeader]
geometry=@ByteArray(\0\0\0\xff\0\0\0\0\0\0\0\x1\0\0\0\x1\0\0\0\0\x1\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\x2\x8b\0\0\0\x5\x1\x1\0\x1\0\0\0\0\0\0\0\0\0\0\0\0\x64\xff\xff\xff\xff\0\0\0\x81\0\0\0\0\0\0\0\x5\0\0\0\x7f\0\0\0\x1\0\0\0\0\0\0\0\xdc\0\0\0\x1\0\0\0\0\0\0\0\x64\0\0\0\x1\0\0\0\0\0\0\0\x64\0\0\0\x1\0\0\0\0\0\0\0h\0\0\0\x1\0\0\0\0\0\0\x3\xe8\0\0\0\0\x64)

[Settings]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x3\0\0\0\0\x1T\0\0\0\x1f\0\0\x4\n\0\0\x2\x17\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16\0\0\0\x1\0\0\0\0\x5P\0\0\x1U\0\0\0:\0\0\x4\t\0\0\x2\x16)

**Comment by jnweiger** [comment]
@eduardoramalho It is still unclear to me, how to reproduce this situation.
Could you attempt to list the steps needed to reproduce, please?
